features/createUser.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Create user
+  As an external brand user
+  I want to be able to retrieve a token for a user
+  So that the user will be able to use the API
+
+  Scenario: Creating user
+    Given I have a users name
+        And I have a users id
```

Comment:
On purpose. The `And` is also a 'Given`. I am not too used to writing gherkin though.